### ISSUE TITLE
max_results attribute cap at 20

### ISSUE THREAD

**Issue by Sarenne** [joetats/youtube_search#19]
Nice library, thanks for releasing this! The `max_results` flag doesn't seem to be able to return more than 20 results.

**Comment by tloki** [comment]
Indeed, as you can see in short code, it only parses HTML, but does not emulate "scrolling" to produce more results...

**Comment by joetats** [comment]
Yes, this is due to pagination. I have an idea on how to get around this by sending the calls, so let me try that out

**Comment by virajkarandikar** [comment]
waiting for this feature!

**Comment by s0501007** [comment]
Still open issue :(

**Comment by solomsk** [comment]
Still open issue :(
